<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_catalog">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 255, 127);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Каталог рецептов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dobavitrez">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(85, 170, 255);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Добавление рецепта</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_out">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>0</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 0px solid;</string>
    </property>
    <property name="text">
     <string>Выйти из аккаунта</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
